app/components/data-download.tsx

```diff
@@ -126,23 +128,39 @@ const prepareData = ({
         }
       }
     }
-
     return 0;
   });
 
   const parsedData = sortedData.map((obs) => {
-    return Object.keys(obs).reduce<Observation>((acc, key) => {
+    const formattedObs = Object.keys(obs).reduce<Observation>((acc, key) => {
+      return {
+        ...acc,
+        [key]: formatIdentity(obs[key]),
+      };
+    }, {});
+
+    return Object.keys(formattedObs).reduce<Observation>((acc, key) => {
       const col = columns[key];
       const parser = dimensionParsers[key];
 
-      return col
-        ? {
-            ...acc,
-            ...{ [makeColumnLabel(col)]: parser(obs[key] as string) },
-          }
-        : acc;
+      if (!col) return acc;
+
+      const value = formattedObs[key];
+      let parsedValue;
+
+      if (value?.toString() === "–") {
+        parsedValue = "-";
```

Comment:
Nit: Would be nice to extract "-" to a constant in the formatters file, so we don't need to go to `formatIdentity` function definition in another file and see that it returns a hard-coded "-" in order to make this comparison. What do you think?

Reply:
sure!